src/include/ipc/dai.h

```diff
@@ -68,6 +68,9 @@ enum sof_ipc_dai_type {
 	SOF_DAI_INTEL_ALH,		/**< Intel ALH */
 	SOF_DAI_IMX_SAI,                /**< i.MX SAI */
 	SOF_DAI_IMX_ESAI,               /**< i.MX ESAI */
+	SOF_DAI_AMD_BT,			/**< Amd BT */
+	SOF_DAI_AMD_SP,			/**< Amd SP */
+	SOF_DAI_AMD_DMIC		/**< Amd DMIC */
```

Comment:
ABI change, this should be tagged as such

Reply:
@balakishorepati this also means we need the same change in the Linux kernel too. i.e. to align the headers.

Reply:
yes same changes has be done in Linux kernel headers.

Reply:
> ABI change, this should be tagged as such
Is there a need to change ABI version ? As per my understanding we don't need to as we didn't change any thing with respect to IPC interface or IPC structures.
Could you also provide more details on tagging the ABI, what needs to be done exactly?

Reply:
@bhiregoudar I've added this in the ABI tool, this just helps us align the kernel drivers with the FW. See https://github.com/orgs/thesofproject/projects/2 we need to do the same for your kernel PR. This will mean AMD support will appear with ABI 3.19

Reply:
Kernel https://github.com/thesofproject/linux/pull/3054 aligned with this PR for ABI 3.19